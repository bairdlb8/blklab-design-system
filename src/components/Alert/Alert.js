import React from 'react'
import PropTypes from 'prop-types'

import './Alert.scss'

const Alert = ({ className = '' }) => {
  return <div role="alert">Alert</div>
}

Alert.propTypes = {}

export default Alert
