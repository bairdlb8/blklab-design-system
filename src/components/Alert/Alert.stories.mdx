import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Alert from './Alert'

<Meta title="Components / Alert" component={ Alert } />

# Alert

## Add some text here

export const Template = (args) => {
  return <Alert {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Alert } />
