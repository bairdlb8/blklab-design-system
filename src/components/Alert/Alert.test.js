import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { toHaveNoViolations } from 'jest-axe'
import Alert from './Alert'

expect.extend(toHaveNoViolations)

describe('<Alert />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Alert />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Alert />)

    expect(asFragment(<Alert />)).toMatchSnapshot()
  })

})
