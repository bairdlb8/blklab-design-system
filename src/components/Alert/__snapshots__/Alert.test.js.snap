// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert /> should take a snapshot 1`] = `
<DocumentFragment>
  <div
    role="alert"
  >
    Alert
  </div>
</DocumentFragment>
`;
